tempodb/backend/gcs/config.go

```diff
@@ -1,12 +1,16 @@
 package gcs
 
-import "time"
+import (
+	"time"
+)
 
 type Config struct {
-	BucketName        string        `yaml:"bucket_name"`
-	ChunkBufferSize   int           `yaml:"chunk_buffer_size"`
-	Endpoint          string        `yaml:"endpoint"`
-	Insecure          bool          `yaml:"insecure"`
-	HedgeRequestsAt   time.Duration `yaml:"hedge_requests_at"`
-	HedgeRequestsUpTo int           `yaml:"hedge_requests_up_to"`
+	BucketName        string            `yaml:"bucket_name"`
+	CacheControl      string            `yaml:"cache_control"`
```

Comment:
i think we should either make these a subject object or prefix them to clarify to the user that they are related to the objects.

Something like?
```
object_cache_control
object_metadata
```